runtimes/ruby/ruby-on-rails.md

```diff
@@ -43,17 +44,15 @@ Then, you need to provide a Gemfile.lock. To do that, please run:
 .../myapp $ git add Gemfile.lock
 .../myapp $ git commit -m "Add production dependency file"
 ```
+## Environment injection
 
-## Choose ruby version
-
-Simply add this to your Gemfile
+To access your variable in your application, nothing simpler! Just get
+it from your environment, like you would with `PATH`:
+`ENV["MY_VARIABLE"]`.
 
-```
-ruby '2.3'
-```
+### Choose ruby version
```

Comment:
To me it should still be a second level title, so it can be listed in the summary at the top of the documentation page